sideboard/run_mainloop.py

```diff
@@ -1,6 +1,28 @@
 from __future__ import unicode_literals
```

Comment:
MAGFest doesn't run anything in daemon mode at the moment, but might care about this if we start, e.g. if we ever split emails into a different process on a different VM.